scout/server/blueprints/variant/templates/variant/variant_details.html

```diff
@@ -253,7 +253,7 @@
           <span class="float-right">{{ variant.spidex_human }}</span>
         </li>
         <li class="list-group-item">
-          <a hfref="https://github.com/Illumina/SpliceAI">SpliceAI</a> DS max
+          <a href="https://doi.org/10.1016/j.cell.2018.12.015">SpliceAI</a> <a href="https://github.com/Illumina/SpliceAI">DS max</a>
```

Comment:
It's working fine! Consider to open the link in a new tab. But I see that the other links (sift, revel, etc) are opening on the same window so it's fine to keep it like this (or to make them all open on new tabs)

Reply:
hfref :)